docs/devices/sensors/MFC.md

```diff
@@ -0,0 +1,28 @@
+# Bronkhorst El-flow mass flow controller
+
+## Introduction
+
+Bronkhorst El-flow mass flow controller (MFC) device driver.
+
+More information about the device can be found in the supplier platform [Bronkhorst](https://www.bronkhorst.com/en-gb/).
+
+## mass flow controller
+
+
+
+## Configuration
+Configuration sample showing all possible parameters:
+
+```toml
+[device.my-MFC]  # This is the MFC identifier
+port = "COM4"    # Access port (serial)
+channel = 1      
+address = 0x80
```

Comment:
check type

Reply:
It would be interesting if these other parameters were described. (channel and address)